README.md

```diff
@@ -107,6 +107,14 @@ Use Skatepark to develop, test, and validate that your program will operate as e
 > [!IMPORTANT]
 > The client program is an _example_ for how to interact with the Managed Structure's API. It is useful for testing your Managed Structure locally, but ultimately you will want to integrate your Managed Structure with your own application. 
 
+
+# Simulating Structure Run Delay
+By default, Skatepark immediately transitions Structure Runs from the `QUEUED` state to the `RUNNING` state.
+If you need artificially create a delay in order to test the behavior of your Structure when it is in the `QUEUED` state, you can do so by setting the `GT_CLOUD_RUN_DELAY` environment variable.
```

Comment:
Open question: instead of calling this a "Run Delay" should this really be a "Queue Time" or "Queue Time Emulation" (QTE lol)?